Constants.js

```diff
@@ -1,4 +1,13 @@
 export default {
-  ROOT_URL: 'https://peoplepower-node.herokuapp.com',
-  PDF_DELETE_DELAY: 20
+  // Bill Generation
+  SERVER_URL: 'https://peoplepower-node.herokuapp.com',
+  PDF_DELETE_DELAY: 20,
+
+  // Mailing
+  SENDER_NAME: 'People Power Solar',
+  EMAIL_LOGO_SRC: 'https://i.imgur.com/wD9o1iM.png',
+
+  // Pledge Invites
+  INVITE_BASE_URL: 'https://peoplepower.netlify.com/onboarding',
+  INVITE_PARAM: 'invite'
```

Comment:
Need to standardize to either `invite` or `token`

Reply:
invite!